src/components/ButtonGroup.tsx

```diff
@@ -73,7 +75,9 @@ function GroupButton(props: GroupButtonProps) {
             }}
             {...tid[defaultTestId(text ?? icon ?? "button")]}
           >
-            {icon && <Icon icon={icon} />}
+            {icon && (
+              <Icon xss={{ marginRight: "4px" }} icon={icon} color={disabled ? undefined : iconColor} inc={iconInc} />
```

Comment:
<img width="378" alt="image" src="https://github.com/homebound-team/beam/assets/20718430/be4f45b2-d0d0-486a-92f6-c0c1dc332df5">

Reply:
Looking at the stories, this margin right has made icon-only button groups look off center? Should it be conditional, only if there is text?

Reply:
`Css.if(text).mrPx(4).$` :-)